ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.applovin.mediation;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import com.applovin.sdk.AppLovinErrorCodes;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.sdk.AppLovinSdkSettings;
+import com.google.android.gms.ads.AdRequest;
+
+/**
+ * Created by thomasso on 1/25/18.
+ */
+
+class AppLovinUtils
+{
+    private static final String DEFAULT_ZONE = "";
+
+    /**
+     * Keys for retrieving values from the server parameters.
+     */
+    private static class ServerParameterKeys
+    {
+        private static final String SDK_KEY   = "sdkKey";
+        private static final String PLACEMENT = "placement";
+    }
+
+    /**
+     * Retrieves the appropriate instance of AppLovin's SDK from the SDK key given in the server parameters, or Android Manifest.
+     */
+    static AppLovinSdk retrieveSdk(Bundle serverParameters, Context context)
+    {
+        final String sdkKey = serverParameters.getString( ServerParameterKeys.SDK_KEY );
```

Comment:
Check for `null`.

Reply:
According to their [docs](https://developers.google.com/admob/android/rewarded-video-adapters), `serverParameters` is actually never going to be `null`, unlike `networkExtras`. If it is, there is something more wrong with AdMob SDK and a crash is inevitable!!

Reply:
Good to know! Same goes for L57

Reply:
Ah, I shall remove thy inconsistency!